### PR TITLE
DDPB-3749 - Update ALB Response Alarms to static threshold

### PR DESCRIPTION
**Pull request by sixdaysandy**
## Purpose
Reduce the amount of alerting spam from ALB Response Time alarms which were tracking anomaly detection, change this to the static threshold of 1 second as agreed with the team.
Fixes DDPB-3749

I have also refactored the PR Pipeline to plan against shared development and the pr environment whilst the PR is building to give quicker (and in the case of shared missing) feedback on terraform issues.

I also noted that the unit tests were being blocked by deployment, but had no dependency on AWS Infrastructure so I have removed that dependency reducing the time to the integration tests being playable from 24 minutes to 16, a saving of 8 minutes.

### Original PR Flow
image

### Revised PR Flow
image

## Checklist
- [x] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/ADR/tech debt doc) where relevant
- [ ] I have added tests to prove my work
- [ ] The product team have approved these changes

### Frontend
- [ ] I have run an in-browser accessibility test (e.g. WAVE, Lighthouse)
- [ ] There are no deprecated CSS classes noted in the profiler
- [ ] Translations are used and the profiler doesn't identify any missing
- [ ] Any links or buttons added are screen reader friendly and contextually complete

### CODE REVIEW

**Review by jamesrwarren — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,7 +74,7 @@ workflows:
 
       - run-task:
           name: backup environment
-          requires: [ api unit test, client unit test, integration test ]
+          requires: [ integration test ]
           filters: { branches: { ignore: [ main ] } }
           task_name: backup
```
Comment (acsauk) [inline .circleci/config.yml:79]: Probably not part of this PR but I think we should move the backup and restore steps to the weekly test run we do

**Review by acsauk — APPROVED** [review]
All looks good 🙏  for a quieter slack channel going forward.
